apps/app/pages/inactive.js

```diff
@@ -1,25 +1,36 @@
 import { useRouter } from "next/router";
 import { Button } from "antd";
-import ErrorPage from "~/layouts/ErrorPage";
+import ErrorPage from "@coderdojobraga/ui/components/ErrorPage";
+import { withAuth } from "~/components/Auth";
 
 function InactiveActions() {
   const router = useRouter();
 
   return (
-    <>
-      <Button onClick={(_) => router.push("/")} key="home" type="primary">
-        Página Inicial
-      </Button>
-    </>
+    <div className="m-auto mt-5 px-4">
+      <div className="m-auto block w-fit">
+        <Button
+          className="mx-2 border bg-[#722ed1] p-2 text-white"
+          onClick={(_) => router.push("/")}
+          key="home"
+          type="primary"
+        >
+          Página Inicial
+        </Button>
```

Comment:
```suggestion
        <button
          className="mx-2 border bg-[#722ed1] p-2 text-white"
          onClick={() => router.push("/")}
          key="home"
        >
          Página Inicial
        </button>
```

And then you can remove `Button` from `antd`.